packages/web-react/src/components/Dropdown/stories/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context/IconsContext';
+import { Icon } from '../../Icon';
+import Dropdown from '../Dropdown';
+import { SpiritDropdownProps, DropdownRenderProps } from '../../../types';
+// import { Button } from '../../Button';
+
+const Story: ComponentStory<typeof Dropdown> = (args: SpiritDropdownProps) => (
+  <IconsProvider value={icons}>
+    <Dropdown {...args} />
+  </IconsProvider>
+);
+
+Story.args = {
+  id: 'DemoDropdown',
+  renderTrigger: ({ isOpen, trigger: { ...restOf } }: DropdownRenderProps) => (
+    // Missing 'ref' or not working 'innerRef' in Button component !!!
+    <button type="button" {...restOf}>
+      Trigger ({isOpen ? 'open' : 'closed'})
+    </button>
+  ),
+  children: (
+    <>
+      <a href="#info" className="d-flex mb-400">
```

Comment:
Missing styles might be solved by using `ButtonLink` here (and `Button` above) from Spirit since we have it or is there a reason why not to use it?

Reply:
I couldn't find an example of a `ButtonLink` where there is an icon + text, so I'm sticking with what I got. 
However, there should be something like `startIcon` and `endIcon` for the `Button` and `ButtonLink` components and then render everything accordingly.

Like:
`<Button startIcon={<SvgIcon />} endIcon={<SvgIcon />}>Button text</Button>`